site/content/docs/user/rootless.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Running kind with Rootless Docker"
+menu:
+  main:
+    parent: "user"
+    identifier: "rootless"
+    weight: 3
+---
+Starting with kind 0.11.0 and Docker 20.10, Rootless Docker can be used as the node provider of kind.
```

Comment:
Should we link to the docker rootless install docs?
https://docs.docker.com/engine/security/rootless/